keyboards/phantagom/baragon/keymaps/default/keymap.c

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 <dennis@kruyt.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+enum Layer {
+    _NUMPAD = 0,      // Standard QWERTY layer
+};
+#define _NP _NUMPAD
```

Comment:
You aren't using these, either remove them or make as many aliases as layers the keymap has, and use them instead of hardcoding numbers
```suggestion
```